### DISCUSSION TITLE
Equivalent of RichTextBox Flow Document in UNo

### DISCUSSION THREAD

**Discussion by HowardPWeiss** [unoplatform/uno#19030] [Q&A]
In my WPF project, I have a rich text box which I use to display and scroll status text generated by my application

``<RichTextBox Name="rtfActions" Height="552" Width="716" TabIndex="10" Margin="907,243,49,44" HorizontalAlignment="Left" VerticalAlignment="Top" >
    <FlowDocument>
         <Paragraph>
             <Run></Run>
         </Paragraph>
     </FlowDocument>
</RichTextBox>

Note that the document gets larger as more text is written to it.  This suggests that embedding a text box in a scrolllist view is not the way to go

What is the equivalent implementation in Uno?

Howard Weiss

**Comment by HowardPWeiss** [comment]
BTW: I am writing an application for C# Linux Visual Studio Code 

Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.6 LTS
Release:	20.04
Codename:	focal
.

**Reply by jeromelaban** [reply]
Thanks for the question. At this time https://github.com/unoplatform/uno/issues/3848 is not supported on Skia targets.

**Comment by lextm** [comment]
`LeXtudio.Windows` NuGet package ships WPF origin controls like RichTextBox and DataGrid to WinUI 3 and Uno Platform, so you might want to give it a try

https://github.com/lextudio/WindowsShims
